src/Mono.Android/Mono.Android.targets

```diff
@@ -153,6 +153,16 @@
        Overwrite="True"
    />
   </Target>
+  <Target Name="_GenerateCodeBehindSources"
```

Comment:
Instead of this target, I would prefer that you instead place `CodeBehind/LayoutBinding.cs` into [`src/Xamarin.Android.Build.Tasks/Resources`](https://github.com/xamarin/xamarin-android/tree/master/src/Xamarin.Android.Build.Tasks/Resources) and have [`Xamarin.Android.Build.Tasks.targets`](https://github.com/xamarin/xamarin-android/blob/master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets) copy the file into the output directory.